addons/website/static/src/js/website.editor.js

```diff
@@ -216,7 +216,7 @@ widget.LinkDialog.include({
 /**
  * Display a welcome message on the homepage when it is empty and that the user is connected.
  */
-base.ready().then(function () {
+$.when(def, base.ready()).then(function () {
```

Comment:
Isn't all ajax.loadXML supposed to be included in base.ready()? I think this file changes are useless.